packages/framework/src/Controller/Admin/ProductPickerController.php

```diff
@@ -63,12 +63,18 @@ public function __construct(
     }
 
     /**
-     * @Route("/product-picker/pick-multiple/{jsInstanceId}/")
+     * @Route("/product-picker/pick-multiple/{jsInstanceId}/{allowMainVariants}/{allowVariants}")
      * @param \Symfony\Component\HttpFoundation\Request $request
      * @param string $jsInstanceId
+     * @param bool $allowMainVariants
+     * @param bool $allowVariants
      */
-    public function pickMultipleAction(Request $request, $jsInstanceId)
-    {
+    public function pickMultipleAction(
```

Comment:
This will not work with strict_types = 1, you are right now changing type, please update it

Reply:
eh? I do not understaand this. What type am I changing?

On other hand here is not used `strict_types=1`.

Reply:
Actually even with strict_types it workl like a charm. I am leaving it as is.

Reply:
ok